backend/src/apiserver/resource/resource_manager.go

```diff
@@ -31,7 +31,7 @@ import (
 	scheduledworkflow "github.com/kubeflow/pipelines/backend/src/crd/pkg/apis/scheduledworkflow/v1alpha1"
 	scheduledworkflowclient "github.com/kubeflow/pipelines/backend/src/crd/pkg/client/clientset/versioned/typed/scheduledworkflow/v1alpha1"
 	"github.com/pkg/errors"
-	"k8s.io/apimachinery/pkg/apis/meta/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
Nit: can remove explicit name since it's the same.

Reply:
vscode keeps adding it back with autoformat. I can remove it, but no guarantees it won't pop back up in the future haha.